playgrounds/angular/project.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Minor commment / Quiestion.

Why is the angular config file new? 
What happened with the old version?

Reply:
With the old version, the application was not updated when the libraries changed.